modules/pytket-quantinuum/pytket/extensions/quantinuum/backends/quantinuum.py

```diff
@@ -372,6 +372,7 @@ def process_circuits(
         * `close_batch`: boolean flag to close the batch after the last circuit,
            default=True.
         * `wasm_file_handler`: a ``WasmFileHandler`` object for linked WASM module.
+
```

Comment:
Is this line needed?

Reply:
Yes, if it is missing the doc build complains... not sure why...

Reply:
Might be possible to replace the empty line by an `\n` at the end of the line above, if you would prefer that I could try.